plugin-name/includes/class-plugin-name.php

```diff
@@ -75,13 +75,19 @@ class Plugin_Name {
 	 * @since    1.0.0
 	 */
 	public function __construct() {
+
 		if ( defined( 'PLUGIN_NAME_VERSION' ) ) {
+
 			$this->version = PLUGIN_NAME_VERSION;
+
 		} else {
+
 			$this->version = '1.0.0';
+
 		}
-		$this->plugin_name = 'plugin-name';
-		$this->plugin_prefix = 'pfx_';
+
+		$this->plugin_name      = 'plugin-name';
+		$this->plugin_prefix    = 'pfx_';
```

Comment:
Christ on a bike. I can't believe it.
LOLOLOLOL. Fixing. But heavily disagreeing here
```
$this->plugin_name = 'plugin-name';
$this->plugin_prefix = 'pfx_';
```
This looks good on GIT.
It does not on a code editor, see screenshot. How does GIT manage to squeeze 13 characters into the same space of 11 ??????
That is just wrong. And since the characters are NOT the same amount, the = sign after it will be displaced on local (see screen)
<img width="469" alt="Screenshot 2021-07-14 at 15 49 19" src="https://user-images.githubusercontent.com/11800236/125593038-e39f90ed-bd0f-4652-8a23-9fffa10d30f9.png">